Makefile

```diff
@@ -0,0 +1,28 @@
+.DEFAULT_GOAL := help
+
+WORKING_DIR := ooyala_player
+JS_TARGET := $(WORKING_DIR)/public/js/translations
+EXTRACT_DIR := $(WORKING_DIR)/translations/en/LC_MESSAGES
+EXTRACTED_DJANGO := $(EXTRACT_DIR)/django-partial.po
+EXTRACTED_TEXT := $(EXTRACT_DIR)/text.po
+EXTRACTED_DJANGOJS := $(EXTRACT_DIR)/djangojs-partial.po
+EXTRACTED_TEXTJS := $(EXTRACT_DIR)/textjs.po
+
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+extract_translations: ## extract strings to be translated, outputting .po files
```

Comment:
Please add instructions in readme file for extracting translations. Same as https://github.com/open-craft/xblock-chat#translation-i18n